content/blog/elm-review-v2/index.md

```diff
@@ -0,0 +1,269 @@
+---
+title: elm-review v2!
+date: '2020-04-30T00:00:00.000Z'
+---
+
+Today I am very excited to release `elm-review` 2.0.0 and to share its new features!
+
+tl;dr: Here is the list of the introduced features:
+
+#### New review capabilities!
+
+- Project rules!
+- Reporting errors for `elm.json`
+- Reading the dependencies' `docs.json` file
```

Comment:
```suggestion
- Full knowledge of direct and indirect dependencies
```
As someone who has no idea what `docs.json` does, something like this might be more helpful 🙂

Reply:
Yeah, that makes sense. It's not "full" though, since you can't read the implementation details, you can just access the publicly exposed information. What you wrote sounds good though :stuck_out_tongue: 

I'll think of other wordings :+1: